src/Compilers/Core/Portable/Compilation/TypeInfo.cs

```diff
@@ -17,12 +17,24 @@ public struct TypeInfo : IEquatable<TypeInfo>
         /// </summary>
         public ITypeSymbol Type { get; }
 
+        /// <summary>
+        /// The inferred nullability of the expression represented by the syntax node.
+        /// </summary>
+        public Nullability Nullability { get; }
```

Comment:
is there ever any concern that the 'Type' vs the 'ConvertedType' might have different Nullability?  Or are tey guaranteed to be the same?

Reply:
Lol.  i can't read.  sorry.